oauth-rest/oauth-resource-server/src/test/java/com/baeldung/resource/ImplicitFlowLiveTest.java

```diff
@@ -16,55 +16,61 @@
 //Before running this live test make sure both authorization server and resource server are running   
 
 public class ImplicitFlowLiveTest {
-	private final static String AUTH_SERVER = "http://localhost:8083/auth/realms/baeldung/protocol/openid-connect";
-	private final static String RESOURCE_SERVER = "http://localhost:8081/resource-server";
-	private final static String REDIRECT_URL = "http://localhost:8082/new-client/login/oauth2/code/custom";
-	private final static String CLIENT_ID = "newClient";
-	private final static String USERNAME = "john@test.com";
-	private final static String PASSWORD = "123";
-	
-
-	@Test
-	public void givenUser_whenUseFooClient_thenOkForFooResourceOnly() {
-		final String accessToken = obtainAccessToken(CLIENT_ID, USERNAME, PASSWORD);
-
-		final Response fooResponse = RestAssured.given().header("Authorization", "Bearer " + accessToken)
```

Comment:
if there are no changes in this file, can we revert this?

Reply:
changes done